code/datums/brain_damage/brain_trauma.dm

```diff
@@ -0,0 +1,49 @@
+//Brain Traumas are the new actual brain damage. Brain damage itself acts as a way to acquire traumas: every time brain damage is dealt, there's a chance of receiving a trauma.
+//This chance gets higher the higher the mob's brainloss is. Removing traumas is a separate thing from removing brain damage: you can get restored to full brain operativity,
+//but keep the quirks, until repaired by mannitol (for mild/special ones) or brain surgery (for severe ones).
+/datum/brain_trauma
```

Comment:
any reason these aren't an extension of status effects?

Reply:
They're very different, traumas are permanent until cured, so they don't really need the status effect timer system, react to some mob actions (when speaking, when hearing something, etc.), have to be transferrable through the brain when it's removed from the body, have to be transferred when monkeying/unmonkeying, and some other stuff i can't remember at the moment.

Reply:
This is more of an argument to be made for status effects to move to components tbh